src/core/include/units/symbol_text.h

```diff
@@ -146,7 +146,7 @@ struct basic_symbol_text {
   [[nodiscard]] friend constexpr auto operator<=>(const basic_symbol_text& lhs,
                                                   const basic_symbol_text<StandardCharT2, N2, M2>& rhs) noexcept
   {
-    if (const auto cmp = lhs.standard() <=> rhs.standard(); cmp != 0) return cmp;
+    if (const auto cmp = lhs.standard() <=> rhs.standard(); cmp != nullptr) return cmp;
```

Comment:
This is UB: https://eel.is/c++draft/cmp.categories#pre-3.

Reply:
See my proposal for another change.

Reply:
Done.